src/pages/projects-page/ProjectAddSubmitButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Button, ButtonProps } from '@material-ui/core';
+import { FragmentLens } from 'src/shared/forminator/core/fragment-lens/FragmentLens';
+import { useFragmentContext } from 'src/shared/forminator/core/fragment/FragmentContext';
+import { useSubmitContext } from 'src/shared/forminator/core/submit/SubmitContext';
+
+interface OwnProps<V> extends ButtonProps {
+  lens: FragmentLens<V>;
+}
+
+type Props<V> = React.PropsWithChildren<OwnProps<V>>;
+
+export function ProjectAddSubmitButton<V>(props: Props<V>) {
+  const { lens, ...restProps } = props;
+  const onSubmit = useSubmitContext();
+  const fragment = useFragmentContext();
```

Comment:
Do not get the fragment from context. You can get the fragment from `substore.fragment`.